app/views/locations/new.erb

```diff
@@ -6,14 +6,14 @@
   <%= f.govuk_error_summary %>
   <div class='govuk-grid-row'>
     <div class='govuk-grid-column-full'>
-      <%= link_to 'Back to list', ips_path, class: 'govuk-back-link' %>
       <h1 class='govuk-heading-l'>Add a location</h1>
     </div>
     <div class='govuk-grid-column-full'>
       <%= f.govuk_text_field :address,
                               hint: { text: "For example, 10 High Street, London" } %>
       <%= f.govuk_text_field :postcode %>
-      <%= f.govuk_submit "Add location" %>
+        <%= f.govuk_submit "Add location" %>
```

Comment:
I think the indentation is slightly off here.

Reply:
Thanks. Odd the linter doesn't catch that.